istioctl/pkg/authz/analyzer.go

```diff
@@ -15,7 +15,7 @@
 // The auth package provides support for checking the authentication and authorization policy applied
```

Comment:
Just curious, do we want to move this somewhere else or at least create a owner file for security owners to own this `pkg/auth` path?

Reply:
Yeah, we could later also move it to the security directory like we did for the converter.